pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -13,10 +13,10 @@
 #define HTTP2_MAX_FRAMES_FOR_EOS_PARSER (HTTP2_MAX_FRAMES_FOR_EOS_PARSER_PER_TAIL_CALL * HTTP2_MAX_TAIL_CALLS_FOR_EOS_PARSER)
 
 // Represents the maximum number of frames we'll process in a single tail call in `handle_headers_frames` program.
-#define HTTP2_MAX_FRAMES_FOR_HEADERS_PARSER_PER_TAIL_CALL 19
+#define HTTP2_MAX_FRAMES_FOR_HEADERS_PARSER_PER_TAIL_CALL 18
 // Represents the maximum number of tail calls to process headers frames.
-// Currently we have up to 240 frames in a packet, thus 13 (13*19 = 247) tail calls is enough.
-#define HTTP2_MAX_TAIL_CALLS_FOR_HEADERS_PARSER 13
+// Currently we have up to 240 frames in a packet, thus 14 (14*18 = 252) tail calls is enough.
```

Comment:
Probably worth documenting why this needed to change, because based on the comment it should've worked.

Reply:
Irrelevant here. The HTTP2 programs are very heavy (nested loops) so we split them to multiple tail calls, each processing multiple HTTP2 frames. The change of Amit was to reduce the number of frames we process in a single tail call to reduce the pressure from the verifier.